nixos/modules/security/acme.nix

```diff
@@ -185,12 +185,15 @@ in
                   path = [ pkgs.simp_le ];
                   preStart = ''
                     mkdir -p '${cfg.directory}'
-                    chown -R '${data.user}:${data.group}' '${cfg.directory}'
+                    chown root:root '${cfg.directory}'
```

Comment:
I'm not sure `root:root` is the right user:group permission for ACME. It runs under root by default, but should the directory also be `root:root`?